CONTRIBUTING.md

```
@@ -0,0 +1,120 @@
+## Steps format
+
+A consistent steps format is essential for the readability and maintainability
+of tests. Follow these guidelines:
+
+- **General Guidelines**:
+  - Avoid using regular expressions to define a step definition. Use tuple
+    format instead for better clarity and maintainability.
+  - Use descriptive placeholder names to help users quickly understand the
+    expected value: `content_type` instead of `type`.
+  - Use `the following` for tabled content.
+  - For anything identified by a property, use `with`: <code>Then the link :
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: code

(MD033, no-inline-html)